pkg/builder/kaniko/kaniko.go

```diff
@@ -31,7 +31,6 @@ const (
 
 type Kaniko struct {
 	system.SystemDependencies
-	ContentName string // Name of the content pushed to Minio
```

Comment:
Why did we remove this?

Reply:
concurrency issues. it is moved to somewhere else, where it is used.